src/ADNLPProblems/cragglvy2.jl

```diff
@@ -12,6 +17,22 @@ function cragglvy2(; n::Int = default_nvar, type::Type{T} = Float64, kwargs...)
       (x[2 * i + 2] - 1)^2 for i = 1:(div(n, 2) - 1)
     )
   end
-  x0 = vcat(1, 2 * ones(T, n - 1))
+  x0 = vcat(T(1), T(2) * ones(T, n - 1))
```

Comment:
```suggestion
  x0 = vcat(1, 2 * ones(T, n - 1))
```
Why is the cast needed?